app/components/forms/events/view/videoroom-form.hbs

```diff
@@ -2,45 +2,51 @@
   <div class="ui grid">
     <div class="ui left aligned sixteen wide mobile eight wide computer column">
       <h3 class="ui header">
-        {{t 'Video Channel'}}
+        {{t "Video Channel"}}
```

Comment:
Why making unnecessary changes?

Reply:
Sir i used prettier for indentation which by default changes ' ->".

Reply:
Revert it